app/GroupMember.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class GroupMember extends Model
```

Comment:
I don't think we need a GroupMember class. If we're going by a DB schema we can have a group_id and then have associated user_id's. We treat the group as an object and members of the group as just users.